data_model/src/transaction.rs

```diff
@@ -1046,6 +1104,31 @@ pub mod error {
             pub time_to_live_ms: u64,
         }
 
+        /// Account does not exist error
+        #[derive(
+            Debug,
+            Display,
+            Clone,
+            PartialEq,
+            Eq,
+            Hash,
+            Decode,
+            Encode,
+            Deserialize,
+            Serialize,
+            IntoSchema,
+        )]
+        #[ffi_type(opaque)]
+        #[cfg_attr(feature = "std", derive(thiserror::Error))]
+        pub enum AccountDoesNotExistError {
```

Comment:
we already have `FindError`. Can we use it please?

Reply:
I used it at the beginning but then decided to create another type, because `FindError` contains too much variants which are meaningless in this place

Reply:
I would use `FindError`. It's not unusual to have a big error variant that encompasses all possible errors but which cannot occur for some system. Just take a look at `std::io::ErrorKind`, it contains errors for both network and filesystem and these don't overlap entirely

Reply:
and it's easier for the user to be dealing with a smaller number of items

Reply:
> It's not unusual to have a big error variant that encompasses all possible errors but which cannot occur for some system.

Yeah, I know, but I'm just not a fan of that approach. I'd better do something like [this](https://sabrinajewson.org/blog/errors) (although it's too verbose)

Reply:
I would vote for `FindError`, because otherwise we bring additional types into the schema. 
And as i see it we use `TransactionRejectionReason` for error reporting not error handling, so redundant variants doesn't seem to be huge deal.

Reply:
`FindError` is my preference as well. There might be need for special handling of accounts after the new schema is introduced, which I'll do later.

Reply:
Done